packages/nouns-webapp/src/pages/Governance/index.tsx

```diff
@@ -1,23 +1,83 @@
-import { Col } from 'react-bootstrap';
+import { Col, Row } from 'react-bootstrap';
 import Section from '../../layout/Section';
 import { useAllProposals, useProposalThreshold } from '../../wrappers/nounsDao';
 import Proposals from '../../components/Proposals';
 import classes from './Governance.module.css';
+import { utils } from 'ethers/lib/ethers';
+import { useEffect, useState } from 'react';
+import clsx from 'clsx';
+import { useTreasuryBalance } from '../../hooks/useTreasuryBalance';
 
 const GovernancePage = () => {
   const { data: proposals } = useAllProposals();
   const threshold = useProposalThreshold();
   const nounsRequired = threshold !== undefined ? threshold + 1 : '...';
   const nounThresholdCopy = `${nounsRequired} ${threshold === 0 ? 'Noun' : 'Nouns'}`;
 
+  const treasuryBalance = useTreasuryBalance();
+  const [treasuryBalanceUSD, setTreasuryBalanceUSD] = useState(0);
+  const [ethUSDConversionRate, setEthUSDConversionRate] = useState(0);
+
+  useEffect(() => {
```

Comment:
Given that this sort of utility will likely be used elsewhere in the app, why don't we move it to its own module?

We should probably take advantage of `@usedapp/coingecko` here: https://usedapp.readthedocs.io/en/latest/coingecko.html#usecoingeckoprice